From b1f11ce33dcf254c71f776393a6de8c2f993d635 Mon Sep 17 00:00:00 2001
From: Manivannan Sadhasivam <manivannan.sadhasivam@linaro.org>
Date: Tue, 4 Dec 2018 18:22:44 +0530
Subject: [PATCH 36/45] reset: hisilicon: Add HiSilicon HI3670 reset support

HI3670 SoC is architecturally same as the HI3660 SoC. Hence, add the
HI3670 compatible to HI3660 reset driver in order to reuse it.

Signed-off-by: Manivannan Sadhasivam <manivannan.sadhasivam@linaro.org>
---
 drivers/reset/hisilicon/reset-hi3660.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/reset/hisilicon/reset-hi3660.c b/drivers/reset/hisilicon/reset-hi3660.c
index 17d8bb128e6e..1c22f3457ba7 100644
--- a/drivers/reset/hisilicon/reset-hi3660.c
+++ b/drivers/reset/hisilicon/reset-hi3660.c
@@ -103,6 +103,7 @@ static int hi3660_reset_probe(struct platform_device *pdev)
 
 static const struct of_device_id hi3660_reset_match[] = {
 	{ .compatible = "hisilicon,hi3660-reset", },
+	{ .compatible = "hisilicon,hi3670-reset", },
 	{},
 };
 MODULE_DEVICE_TABLE(of, hi3660_reset_match);
-- 
2.15.1

